homeassistant/components/zwave_js/__init__.py

```diff
@@ -55,49 +62,70 @@ async def async_on_initialized():
         LOGGER.info("Connection to Zwave JS Server initialized")
         initialized.set()
 
-        # This should be moved to the Node Ready command or something
-        # And then listen to new nodes too.
-        dev_reg = await device_registry.async_get_registry(hass)
+        # run discovery on all ready nodes
         for node in client.driver.controller.nodes.values():
-            register_node_in_dev_reg(entry, dev_reg, client, node)
+            if not node.ready:
+                continue
+            asyncio.create_task(async_on_node_ready(node))
+        # TODO: register callback for "node_ready" event
 
     async def async_on_disconnect():
         """Handle websocket is disconnected."""
         LOGGER.info("Disconnected from Zwave JS Server")
         # TODO: signal entities to update availability state
 
-    for component in PLATFORMS:
-        hass.async_create_task(
-            hass.config_entries.async_forward_entry_setup(entry, component)
-        )
+    async def async_on_node_ready(node: ZwaveNode):
```

Comment:
Don't forget to type None return value.
```suggestion
    async def async_on_node_ready(node: ZwaveNode) -> None:
```